### ISSUE TITLE
Special accented characters for Spanish (and probably French, German, Italian) missing from MP

### ISSUE THREAD

**Issue by IlDucci** [iortcw/iortcw#144]
Greetings. I recently tried iortcw on Win 10 x64 with my copy of the Spanish Special Edition and while the singleplayer works as intended, it seems that the MP's menus have a character encoding problem or something similar, as all accented characters are gone there.

I'm using the stable 1.51c x64 version.

This is iortcw:
![ioWolfMP 2022-04-12 16-36-38-25](https://user-images.githubusercontent.com/5641681/162989014-5f3f3361-18ea-4fc1-950d-38c56e938b19.png)

And this is a fresh RTCW copy, bought from Steam, with the language option changed in the cfg file:
![20220412163427_1](https://user-images.githubusercontent.com/5641681/162989104-a2d987eb-7a14-4ab1-b833-ad36adfc94f9.jpg)

Notice that the word CRÉDITOS is shown as CRDITOS in the ioWolfMP version.

**Comment by MAN-AT-ARMS** [comment]
These translations are part of the game assets (the pk3 files). SP translation was probably updated while MP was not.
This is not something that can/should be changed in the game engine.

**Comment by IlDucci** [comment]
You misunderstood this. The game assets, as you say, used for both the iortcw and Steam version are exactly the same. What changes is the executable, which seems to be bugged in your case.

**Comment by MAN-AT-ARMS** [comment]
I fixed the issue in 066e86a but it only works when using dynamic libraries. For some reason qvm modules don't like those higher ascii characters. I'm not sure why. That would be a question for @zturtleman 

So, right now you would have to set vm_cgame and vm_ui to 0 to get those characters to display.

**Comment by zturtleman** [comment]
(unsigned char) cast doesn't work in QVMs.
```c
index = (unsigned char)*s;
```
It's still negative so it gets caught in
```c
// NERVE - SMF - don't draw tabs and newlines
if ( index < 20 ) {
	s++;
	count++;
	continue;
}
```
The solution is to use `*s & 255`.

Fixed in 292064b860c490acaaf368a1a67317eacd3b07a7.
